REVIEW.md

```diff
@@ -0,0 +1,192 @@
+    - Are regression tests needed?
+      If a reproducible bug is being fixed in this PR, one may be needed, and reviewers should help the contributor design and implement them.
+      In case the contributor is unable to add the test, the reviewer should do so in a follow-up PR.
+
+ - [ ] Sustainability and maintenance
+    - Is the code easy to read and low on redundancy?
+    - Should variable names be improved for clarity or consistency?
+    - Should comments or docstrings be added or removed?
+    - Will the cost of maintaining a new feature be worth its benefits?
+    - Will the new code add any dependencies on other projects?
+
+- [ ] User-facing changes
+    - What use cases might this change influence?
+    - Does the user experience match existing patterns in Spyder?
+      If it does not, why was the choice made?
+    - Is in-interface text (new or modified) appropriately descriptive and free of spelling or grammar issues that would make its meaning unclear?
+
+#### Optional review
+
+The following are themes that may be helpful to cover on a PR, but are optional.
+
+**Note**: The difference between an objective improvement and a subjective preference isn’t always clear.
+Reviewers should recall that the review process is primarily about reducing risk to the project.
+When reviewing code, one should aim at preventing situations which may introduce a bug, deprecation or regression.
+
+- [ ] Improvements
+    - Could the code easily be rewritten to run much more efficiently for relevant settings?
+    - Is the code backwards compatible with previous versions?
+      (Or is a deprecation cycle necessary?)
+    - Is there feedback that would be helpful for solving future issues surrounding this change?
+      Are there non-urgent concerns that the team should be made aware of?
+    - If a bug is fixed, does the PR include a non-regression test?
+    - Are there images that should be added, resized or optimized?
+
+- [ ] User-facing changes
+    - Does the user interface or other visual elements match existing patterns in Spyder?
+      If it does not, is there a reason?
+
+Review that addresses these questions on a PR will be prefixed with `[optional]` so that the PR author(s) are clear that this feedback is not a blocker.
+Addressing this feedback is considered above-and-beyond, and may be more relevant on some PRs than others.
+PR authors may address these reviews, but are not required to do so in order for the PR to be eligible for merging.
+
+### For documentation/website-focused review
+
+#### Required review
+
+The following are areas the community expects to be covered in the review of a documentation-focused PR.
+In general, reviewers need to consider whether the content is a good fit, accurate and generally clear to readers, and builds and renders with no simmediate user-visible issues.
+
+Review that addresses these questions on a PR will be prefixed with `[required]` so that the PR author(s) are clear that they are blockers.
```

Comment:
```suggestion
```

Just so we all keep track, see the [discussion](https://github.com/spyder-ide/spyder/pull/16789#discussion_r747183383) about this line (and the one above).

Reply:
I haven't included these suggestions because I think they undermine the rest of the guidelines. I mentioned more in the linked discussion.

Reply:
Sorry, I didn't mean to imply you should make this with with the rest, just left this here for tracking purposes if we do decide to implement the proposal there. My mistake; really sorry about the confusion.